react/features/base/participants/middleware.js

```diff
@@ -231,6 +234,57 @@ MiddlewareRegistry.register(store => next => action => {
     case PARTICIPANT_UPDATED:
         return _participantJoinedOrUpdated(store, next, action);
 
+    case OVERWRITE_PARTICIPANT_NAME: {
```

Comment:
Won't dispatching an unmodified `UPDATE_BREAKOUT_ROOMS` action trigger the name update in the other middleware? At least for the breakout rooms branch, that is.

Reply:
I'll add another param to the dispatch to make sure that it doesn't